src/components/Verse/VerseSettingsMenu.tsx

```diff
@@ -0,0 +1,54 @@
+import { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import Verse from '../../../types/Verse';
+import VerseSettingsMenuItem from './VerseSettingsMenuItem';
+import CopyIcon from '../../../public/icons/copy.svg';
+
+interface Props {
+  verse: Verse;
+}
+
+const VerseSettingsMenu: React.FC<Props> = ({ verse }) => {
+  const [copied, setCopied] = useState(false);
+
+  useEffect(() => {
+    let timeoutId: NodeJS.Timeout;
```

Comment:
nit: is there a non nodeJS type definition that we can use?

Reply:
Well the other option is `ReturnType<typeof setTimeout>` but when you hover on it, it still shows `NodeJS.Timeout` since the type that `clearTimeout` accepts is `NodeJs.Timeout`:

`declare function clearTimeout(timeoutId: NodeJS.Timeout): void;`